release-notes/opensearch-ml-common.release-notes-3.0.0.0-alpha1.md

```diff
@@ -0,0 +1,16 @@
+## Version 3.0.0.0-alpha1 Release Notes
+
+Compatible with OpenSearch 3.0.0.0-alpha1
+
+### Bug Fixes
+* fix building error due to a breaking change from core (#3617)[https://github.com/opensearch-project/ml-commons/pull/3617]
+* Support sentence highlighting QA model (#3600)[https://github.com/opensearch-project/ml-commons/pull/3600]
```

Comment:
Should we put this under Enhancements?

Reply:
will move this, thanks!!!

Reply:
Can you also use upper case letters for first letter of each line?
Thanks!